packages/api/core/src/util/install-dependencies.ts

```diff
@@ -14,25 +14,38 @@ export enum DepVersionRestriction {
   RANGE = 'RANGE',
 }
 
+/**
+ *  To install the specified packages as devDependencies
+ * `npm add` and `pnpm add` commands use `--save-dev` option
+ * `yarn add` command use `--dev` option
+ */
+export const getInstallDevDepsOption = (): string => (isNpm() || isPnpm() ? '--save-dev' : '--dev');
+
+/**
+ * To install the specified packages with exact version number instead of version range
+ * `npm add` and `pnpm add` commands use `--save-exact` option
+ * `yarn add` command use `--exact` option
+ */
+export const getInstallExactDepsOption = (): string => (isNpm() || isPnpm() ? '--save-exact' : '--exact');
+
 export default async (dir: string, deps: string[], depType = DepType.PROD, versionRestriction = DepVersionRestriction.RANGE): Promise<void> => {
-  d('installing', JSON.stringify(deps), 'in:', dir, `depType=${depType},versionRestriction=${versionRestriction},withYarn=${hasYarn()}`);
+  d('installing', JSON.stringify(deps), 'in:', dir, `depType=${depType},versionRestriction=${versionRestriction}}`);
```

Comment:
suggestion: let's keep the log line here but indicate which package manager is being used